spec/spec_helper.rb

```diff
@@ -5,15 +5,15 @@
 
 RSpec.configure do |config|
   # DatabaseCleaner
-  config.before(:suite) {
-    DatabaseCleaner.clean_with :deletion, except: ['spree_countries', 'spree_states']
-  }
-  config.before(:each)           { DatabaseCleaner.strategy = :transaction }
-  config.before(:each, concurrency: true) {
+  config.before(:each, concurrency: true) do
+    config.use_transactional_fixtures = false
     DatabaseCleaner.strategy = :deletion, { except: ['spree_countries', 'spree_states'] }
-  }
-  config.before(:each)           { DatabaseCleaner.start }
-  config.after(:each)            { DatabaseCleaner.clean }
+    DatabaseCleaner.start
+  end
+  config.after(:each, concurrency: true) do
```

Comment:
https://github.com/DatabaseCleaner/database_cleaner#rspec-with-capybara-example

> It's also recommended to use `append_after` to ensure DatabaseCleaner.clean runs after the after-test cleanup capybara/rspec installs.

Might be a good idea to update to match.

Reply:
That documentation is outdated but it wouldn't hurt to use `append_after`, I guess. Here I matched our previous behaviour.